docs/topics/coroutine-context-and-dispatchers.md

```diff
@@ -258,8 +270,13 @@ same coroutine, as you can see in the output below:
 
 <!--- TEST -->
 
-Note that this example also uses the `use` function from the Kotlin standard library to release threads
-created with [newSingleThreadContext] when they are no longer needed. 
+The example above uses the [use](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.io/use.html) function from the Kotlin standard library
+to properly release thread resources created by [newSingleThreadContext] when they're no longer needed.
+
+>Note that [withContext] doesn't create a new coroutine -
+>it merely provides a [CoroutineScope] with the specified context.
```

Comment:
That's debatable: https://pl.kotl.in/b9SFlN25v. I would avoid saying this in the documentation.